spec/spec_helper_acceptance.rb

```diff
@@ -79,17 +79,29 @@
           ]
 
           additional_gem_opts = ["--no-ri", "--no-rdoc"]
+          windows_cmd = [ '/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin/ruby.exe', 'C:\Program Files\Puppet Labs\Puppet\sys\ruby\bin\gem' ]
 
           if is_windows?(host)
             # shield the quoting from beaker's autoquoting to correctly quote the space
             # also avoid the gem.bat, as that cannot pass arguments with spaces and other special characters through
-            windows_cmd = [ '/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin/ruby.exe', 'C:\Program Files\Puppet Labs\Puppet\sys\ruby\bin\gem' , 'install' ]
+            if on(host, "grep SSL_CERT_ ~/.bashrc 2>/dev/null", :acceptable_exit_codes => [0,1]).output == ''
```

Comment:
grep can return nothing, which to ruby looks like a failure, even though the absence of it is a signal to us. Ergo, I need to allow it through. Without that ruby will get an exception.